projects/fsastorefrontlib/src/cms-components/message-notification/message-notification.component.ts

```diff
@@ -28,18 +29,20 @@ export class MessageNotificationComponent implements OnInit, OnDestroy {
     protected inboxService: InboxService,
     protected changeDetectorRef: ChangeDetectorRef,
     protected authService: AuthService,
-    protected userService: UserService
+    protected userProfileService: UserProfileService
```

Comment:
Can we inject UserProfileFacade instead, as you did for the account facade?

Reply:
Here, when UserProfileFacade or UserAccountFacade is used functionality is not working and because of that I used service directly.